hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 statehigh:
-  - ttl: 600
-    type: CNAME
-    value: schacks.netlify.com.
+- ttl: 300
+  type: CNAME
+  value: schacks.netlify.com.
 status:
-  - ttl: 600
-    type: CNAME
-    value: checkly-dashboards.com.
+- ttl: 300
+  type: CNAME
+  value: checkly-dashboards.com.
 status.haas:
+  ttl: 300
   type: A
   value: 34.138.52.92
 status.hcb:
-  - ttl: 3600
-    type: CNAME
-    value: cname.appsignal-status.com.
+- ttl: 300
+  type: CNAME
+  value: cname.appsignal-status.com.
 stemegypt:
-  - ttl: 900
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 stemsharkya:
-  - ttl: 600
-    type: CNAME
-    value: stemsharkya.netlify.app.
+- ttl: 300
+  type: CNAME
+  value: stemsharkya.netlify.app.
 stickerhuman:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
-storyboard: 
-  - ttl: 600
-    type: CNAME 
-    value: a.selfhosted.hackclub.com. 
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
+storyboard:
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 straya:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 streaks:
-  - ttl: 600
-    type: CNAME
-    value: adoring-brown-d90455.netlify.app.
+- ttl: 300
+  type: CNAME
+  value: adoring-brown-d90455.netlify.app.
 strong1._domainkey:
+  ttl: 300
   type: CNAME
   value: strong1._domainkey.helpscout.net.
 strong2._domainkey:
+  ttl: 300
   type: CNAME
   value: strong2._domainkey.helpscout.net.
 submit:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 submit.v2:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 suceava:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 2020-summer:
-  - ttl: 600
-    type: ALIAS
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 summer:
-  - ttl: 600
-    type: ALIAS
-    value: a.selfhosted.hackclub.com.
-  - ttl: 600
-    type: TXT
-    values:
-      - google-site-verification=zx9NMOA8WwJ7FlmnASICBEvNzpAMhWSgX2Api6ROjAk
-      - v=spf1 include:_spf.google.com ~all
-  - ttl: 600
-    type: MX
-    values:
-      - exchange: aspmx.l.google.com.
-        preference: 1
-      - exchange: alt1.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt2.aspmx.l.google.com.
-        preference: 5
-      - exchange: aspmx2.googlemail.com.
-        preference: 10
-      - exchange: aspmx3.googlemail.com.
-        preference: 10
-      - exchange: aspmx4.googlemail.com.
-        preference: 10
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?